test/zcl-loader.test.js

```diff
@@ -291,6 +291,8 @@ test(
       expect(x).toEqual(630)
       x = await queryZcl.selectAllAtomics(db, packageId)
       expect(x.length).toEqual(56)
+      x = await queryZcl.selectAllStrings(db, packageId)
+      expect(x.length).toEqual(5) // 5th string is a subatomic type
```

Comment:
Add a test which checks all the strings here to improve tests for x.includes('str'), etc

Reply:
done